From bb7efe14b1dd07d91e93a1015c86d43f3e7c72d4 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Mon, 5 Jan 2015 11:53:07 -0500
Subject: [redhat] kabi: Adding symbol pci_set_master (drivers/pci/pci.c)

Message-id: <e483ad5596ff6c5834b7e258c04ce45a4436914c.1420451770.git.poros@redhat.com>
Patchwork-id: 102439
O-Subject: [PATCH RHEL7.1 10/24] kabi: Adding symbol pci_set_master (drivers/pci/pci.c)
Bugzilla: 1085963
RH-Acked-by: Myron Stowe <mstowe@redhat.com>

Bugzilla: 1085963
    https://bugzilla.redhat.com/show_bug.cgi?id=1085963

    Description:
    ============
    Used to re-enable bus mastering after a driver unload reload.
    Used in hpvsa.ko, hpdsa.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/pci_set_master
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64le/pci_set_master
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/pci_set_master
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/pci_set_master

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/pci_set_master b/redhat/kabi/kabi-module/kabi_ppc64/pci_set_master
new file mode 100644
index 0000000..59c742f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/pci_set_master
@@ -0,0 +1,2 @@
+#1-
+0x8310b417 pci_set_master vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_ppc64le/pci_set_master b/redhat/kabi/kabi-module/kabi_ppc64le/pci_set_master
new file mode 100644
index 0000000..e362204
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64le/pci_set_master
@@ -0,0 +1,2 @@
+#1-
+0xe0642eb4 pci_set_master vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/pci_set_master b/redhat/kabi/kabi-module/kabi_s390x/pci_set_master
new file mode 100644
index 0000000..6584dd4
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/pci_set_master
@@ -0,0 +1,2 @@
+#1-
+0x8820828a pci_set_master vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/pci_set_master b/redhat/kabi/kabi-module/kabi_x86_64/pci_set_master
new file mode 100644
index 0000000..f7e8f36
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/pci_set_master
@@ -0,0 +1,2 @@
+#1-
+0xbe4a1520 pci_set_master vmlinux EXPORT_SYMBOL
-- 
1.7.1